src/datasources/asset/components/editors/list-assets/ListAssetsEditor.tsx

```diff
@@ -36,8 +37,25 @@ export function ListAssetsEditor({ query, handleQueryChange, datasource }: Props
     }
   }
 
+  function onOutputTypeChange(newValue: OutputType) {
+    let updatedQuery: ListAssetsQuery = { ...query };
+    if (query.outputType !== newValue) {
+      updatedQuery = {
+        ...query, outputType: newValue,
+      }
+    }
+    handleQueryChange(updatedQuery, true);
+  }
+
   return (
     <>
+      <InlineField label="Output" labelWidth={22} tooltip={tooltips.listAssets.outputType}>
+        <RadioButtonGroup
+          options={Object.values(OutputType).map(value => ({ label: value, value })) as SelectableValue[]}
```

Comment:
Maybe we could do this in a const as these won't change.
```suggestion
          options={outputTypeOptions}
```